packages/terra-infinite-list/src/terra-dev-site/doc/example/VirtualizedExample.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import InfiniteList, { Item } from 'terra-infinite-list';
-import Placeholder from 'terra-doc-template/lib/Placeholder';
+import { Placeholder } from '@cerner/terra-docs';
```

Comment:
Curious..! Not able to understand the difference between  @cerner/terra-docs and terra-docs..? so would like to know the purpose of adding @ to package name.

Reply:
We're moving to providing all our packages as scoped packages.
Scoped packages allow us to:
* Group all Cerner packages in node modules under @cerner (incidentally it makes these packages easier to find since scoped packages are are the top)
* Automatically associate any published package with the @cerner npm org
* Reduce name conflicts without having to prepend terra-, IE we could fork an npm package and publish under the @cerner scope without fully renaming it.

Babel has made a similar move and scoped all packages under @babel

See npm docs for more on [scoped packages](https://docs.npmjs.com/using-npm/scope.html).